packages/css/src/components/application-header/application-header.scss

```diff
@@ -0,0 +1,3 @@
+.dso-app-heading {
```

Comment:
Bijzonder dat de bestandsnamen `application-header` heten, maar het component `app-heading`. Zullen we dit synchroniseren?

`app-heading` is slecht gekozen, maar laten we daar de API niet op aanpassen. Dat is een misstap. Ik stel voor dat we de bestandsnamen aanpassen naar `app-heading.scss` enz.